.gitignore

```diff
@@ -40,3 +40,15 @@ npm-debug.log
 tools/*.jar
 tools/cloud_sql_proxy
 .DS_Store
+client/py/.pydevproject
+common-api/.gradle/*
+common-api/.idea/*
+common-api/build/*
+common-api/common-api.iml
+public-api/.idea/*
+public-api/.swagger-codegen/*
+public-api/build/*
+public-api/public-api.iml
+public-api/src/generated/
+public-api/src/main/webapp/WEB-INF/appengine-web.xml
+
```

Comment:
sa-key per above? Could we just omit anything called `appengine-web.xml` or sa-key.json?